### PR TITLE
Add support to cjs module format to npm package

### PR DESCRIPTION
**Pull request by davidcostadev**
This pr add support for `cjs` build and improve the publish method.

This will fix the test error too:

**Jest encountered an unexpected token**
```
1: {import 'webrtc-adapter';
    ^^^^^^
```

### CODE REVIEW

**Comment by Mazuh** [comment]
it's actually already published, you're using it on your demo :laughing: but this is indeed a great improvement, we were needing this

super!

**Review by Mazuh — APPROVED** [review]

**Comment by davidcostadev** [comment]
Yeah I know, maybe the title is not the best, I will improve it.

File: .babelrc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          ie: '11'
```
Comment (Mazuh) [inline .babelrc.js]: I guess there's already a target on `package.json` too, remove one of them

Comment (davidcostadev) [reply .babelrc.js]: Yeah, That's true, Thanks.

**Comment by jchavanton** [comment]
since you are already two, this counts as a review, you guys can merge, thanks !

**Comment by Mazuh** [comment]
Done. Published it, I'm imagined you're needing this on your demo.

**Pull request merged**